README.md

```diff
@@ -1,21 +1,111 @@
+        * An optional entry `"conditionals"` allows generating different weights depending on the values of settings that have already been generated (which means the must be above this setting in the `"weights"` array). The entry is an array of objects, each with the following entries:
+            * `"setting"`: The name of the setting whose value to check.
+            * `"conditions"`: An array of values of that setting for which this conditional will be used.
+            * `"values"`: The weights to use if any of the condition is met, in the same format as the base `"values"` entry (which will be used if no conditionals apply).
+
+## Building from source
+
+1. Install Rust:
+    * On Windows, download and run [rustup-init.exe](https://win.rustup.rs/) and follow its instructions.
+    * On other platforms, please see [the Rust website](https://www.rust-lang.org/learn/get-started) for instructions.
+2. (Skip this step if you're not on Windows.) If you're on Windows, you'll also need to download and install [Visual Studio](https://visualstudio.microsoft.com/vs/) (the Community edition should work). On the “Workloads” screen of the installer, make sure “Desktop development with C++” is selected. (Note that [Visual Studio Code](https://code.visualstudio.com/) is not the same thing as Visual Studio. You need VS, not VS Code.)
+3. Install [Python](https://python.org/) version 3.6 or higher, making sure to check the “add to PATH” option.
+4. Download and extract [this repository](https://github.com/matthewkirby/plando-random-settings/archive/master.zip)
+5. Open a command line in the extracted folder:
+    * On Windows, locate the folder in File Explorer, then right-click it while holding shift and select “Open PowerShell window here” or “Open command prompt here”.
+    * On other platforms, open a command line and navigate to the folder using `cd`.
+6. In the command line, run one of the following commands. Depending on your computer, this may take a while.
+
+    To install the command-line version:
+
+    ```sh
+    cargo install --path=crate/rsl-cli
+    ```
+
+    To install the main app:
+
+    ```sh
+    cargo install --path=crate/rsl-gui
+    ```
+7. You can now launch the app by running the command `rsl-gui`, and/or use the command-line version `rsl` (see `rsl --help` for details).
+
+## Releasing a new version
+
+You will need a 64-bit Windows PC and a Mac.
+
+1. If the randomizer version for league seeds should be updated, change the constants `LEAGUE_COMMIT_HASH` and `LEAGUE_VERSION` in `crate/ootr/src/lib.rs` accordingly
+2. Bump the version number in:
+    * `assets/macos/RSL.app/Contents/Info.plist` (also increment `CFBundleVersion`)
```

Comment:
Can you call out both CFBundleShortVersionString and CFBundleVersion? Do you update CFBundleVersion by adding 1?